web/src/services/auth/index.tsx

```diff
@@ -7,17 +7,22 @@ import { useAuthenticationRequired } from "./useAuth";
 export { AuthProvider } from "./authProvider";
 export { useAuth, useCleanUrl, useAuthenticationRequired } from "./useAuth";
 
-export const AuthenticationRequiredPage: React.FC<{ children?: ReactNode }> = ({ children }) => {
+const AuthenticationRequiredPage: React.FC<{ children?: ReactNode }> = ({ children }) => {
   const [isAuthenticated] = useAuthenticationRequired(); // TODO: show error
   return isAuthenticated && children ? <>{children}</> : null;
 };
 
-export const withAuthorisation = (): ((props: any) => React.FC<any>) => {
+const withAuthorisation = (): ((props: any) => React.FC<any>) => {
   const authProvider = window.REEARTH_CONFIG?.authProvider;
-
   if (authProvider === "cognito") {
+    // Check if authProvider is explicitly defined as "cognito"
     return withAuthenticator as unknown as (props: any) => React.FC<any>;
+  } else if (authProvider === "auth0") {
+    // Handle the case when authProvider is undefined
```

Comment:
This is handling if authProvider is "auth0", right?
Though I feel like the if statements speak for themselves, so not sure comments are really necessary here 🤔

Reply:
Done.